api/bolt/helmuserrepository/helmuserrepository.go

```diff
@@ -39,7 +39,7 @@ func (service *Service) HelmUserRepositoryByUserID(userID portainer.UserID) ([]p
 		cursor := bucket.Cursor()
 		for k, v := cursor.First(); k != nil; k, v = cursor.Next() {
 			var record portainer.HelmUserRepository
-			err := internal.UnmarshalObject(v, &record)
+			err := internal.UnmarshalObject(v, &record, service.connection.EncryptionKey)
```

Comment:
I hope we can refactor this so the encryption key doesn't need to be added to every Service - something like putting the marshal functions as calls to `service.connection` - this might become simpler post store refactor merge

Reply:
I copied `json.go` from your commit and did this so that I can demo it today. I didn't like this either.